vars/all_var.yml

```diff
@@ -210,56 +92,12 @@
   db_user_name: "{{ env_db_user_name }}"
   db_user_pw: "{{ env_db_user_pw }}"
   db_platform_type: "postgres"
-  db_engine_driver: "django.db.backends.postgresql_psycopg2"
```

Comment:
This value is not needed to construct the database connection string, or used anywhere else.